Tests/ML/test_config_helpers.py

```diff
@@ -120,3 +125,55 @@ def test_equally_weighted_classes_fails(num_fg_clases: int, background_weight: O
     classes = [""] * num_fg_clases
     with pytest.raises(ValueError):
         equally_weighted_classes(classes, background_weight)
+
+
+def create_dataset_csv(test_output_dirs: OutputFolderForTests) -> Path:
+    """Create dummy dataset csv file for tests,
+    deleting any pre-existing file."""
+    test_csv = "test_dataset.csv"
+    root_dir = test_output_dirs.root_dir
+    dataset_csv_path = root_dir / test_csv
+    if dataset_csv_path.exists():
+        dataset_csv_path.unlink()
+    dataset_csv_path.write_text("""subject,channel,filePath""")
+    return dataset_csv_path
+
+
+def validate_dataset_paths(
+        model_config: Union[ScalarModelBase, SegmentationModelBase]) -> None:
+    """Check that validation of dataset paths is succeeds when csv file exists,
+    and fails when it's missing.""" 
+    ml_util.validate_dataset_paths(model_config.local_dataset,
+                                   model_config.dataset_csv)
+
+    dataset_csv_path = model_config.local_dataset / model_config.dataset_csv
+    dataset_csv_path.unlink()
+
+    ex_message = f"ValueError: The dataset file {model_config.dataset_csv} "\
+                  "is not present"
+    with pytest.raises(ValueError) as ex:
+        ml_util.validate_dataset_paths(model_config.local_dataset,
+                                       model_config.dataset_csv)
+        assert ex_message in str(ex)
```

Comment:
This assert needs to be outside the context manager.

Reply:
With the assert moved outside the context manager, the test unfortunately fails.  The definition of ex_message should be changed to:
```
ex_message = f"The dataset file {model_config.dataset_csv} is not present"
```